addins/Cake.Issues.Testing.yml

```diff
@@ -2,8 +2,12 @@ Name: Cake.Issues.Testing
 NuGet: Cake.Issues.Testing
 Assemblies:
 - "/**/Cake.Issues.Testing.dll"
-Repository: https://github.com/cake-contrib/Cake.Issues.Testing
-Author: BBT Software AG
-Description: "The testing extensions for the Cake Issues Addin provide helpers to implement test cases for Cake Issue Addins."
+Repository: http://cake-contrib.github.io/Cake.Issues.Website/
```

Comment:
From where does this come from? This is the website not the repository

Reply:
If comes from the nuget package metadata: https://www.nuget.org/packages/Cake.Issues.Testing/ Pay attention to the "Project site" link in the top right corner.

Reply:
Here's the reason why: @gep13 and I noticed a while ago that many addins had been moved to the cake-contrib organization but authors had forgotten to update the project URL in their nuspec (the URL still pointed to the repo under their personal github user). Since the audit tool uses this URL to determine whether an addin has ben moved to the cake-contrib or not, it resulted in many many false-positives. Therefore I implemented logic to query the github API and check if a given project is available under the cake-contrib organization. If so, disregard the URL in the nuspec otherwise simply use whatever URL is in the nuspec.

There are 20 or so `Cake.Issues.*` projects under the cake-contrib organization but I don't see `Cake.Issues.Testing`. That explains why the audit tool uses the URL from the nuspec.